cli/cmd/plugin/ui/web/tanzu-ui/src/shared/components/UnmanagedClusterInfo/UnmanagedClusterInfo.tsx

```diff
@@ -0,0 +1,64 @@
+// React imports
+import React from 'react';
+
+// Library imports
+import { CdsButton } from '@cds/react/button';
+import { CdsIcon } from '@cds/react/icon';
+import { CdsDivider } from '@cds/react/divider';
+
+// App imports
+import './UnmanagedClusterInfo.scss';
+
+function UnmanagedClusterInfo(props: any) {
```

Comment:
Very nice that you picked up on using props to pass data to a child component.

We can take this one step further. Instead of defining props as type 'any' what we should do right above this function is define an interface. The interface will define the 'shape' of the data that is expected when props are passed.

So above line 12 you will create that interface using 'export interface UnmanagedClusterInfoProps' and then define the attributes that are expected in that object (name, provider, status). 

Now what will happen is if someone goes to reuse this component, and they do not include required data that you define in that interface, the React compiler will throw an error to warn them.